spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -812,6 +812,16 @@
         "level": "error"
       }
     },
+    {
+      "name": "liquibase.test-rollback-on-update",
```

Comment:
No need to provide deprecation metadata for a property that was introduced after we move the namespace. This entry should be removed.